docs/introduction/Car-registry-tutorial.rst

```diff
@@ -2,73 +2,76 @@
 
-*A:* I create a new Car Sell Order Box that contains the price in coins and information from the Car Entry Box. So cars can exist in the Sidechain as a Car Entry Box or as a Car Sell Order, but not at the same time. Also, this box contains the buyer’s public key. When I create a sell order, Sidechain should check if there is no other active sell order with this Car Entry Box. The current Sell Order consists of the same information that consists of the Car Entry Box plus description.
+*A:* I create a new Car Sell Order Box that contains the price in coins and information from the Car Entry Box. So cars can exist in the Sidechain either as a Car Entry Box or as a Car Sell Order, but not both at the same time. Also, this box contains the buyer’s public key. When I create a sell order, the Sidechain should verify that there is no other active sell order with this Car Entry Box. The current Sell Order consists of the same information that contained in the Car Entry Box plus description (is the description optional?).
 
 3
-**Q: I want to see all available Sell orders in Sidechain**
+**Q: I want to see all available Sell orders in the Sidechain.**
 
 *A:* Have additional storage, which is managed by ApplicationState and stores all Car Sell Orders. All these orders can be retrieved using the new HTTP API call. 
 
-
 4
 **Q: I want to accept a sell order and buy the car.**
 
-*A:* By accepting sell order, I create a new transaction in the Sidechain, which creates a new Car Entry Box with my public key as Proposition and transfers coins amount from me to the previous car owner.
+*A:* By accepting a sell order, I create a new transaction in the Sidechain, which creates a new Car Entry Box with my public key as Proposition and transfers the correct value of coins from me to the seller.
 
 5
 **Q: I want to cancel my Car Sell Order.**
 
-*A:* I create a new transaction containing Car Sell Order as input and Car Entry Box with my public key as Proposition as output.
+*A:* I create a new transaction containing the Car Sell Order as input and Car Entry Box with my public key as Proposition as output.
 
 6.
-**Q: I want to see my car entry boxes and car sell orders related to me (both created by me and proposed to me).**
+**Q: I want to see the car entry boxes and car sell orders related to me (both created by me and proposed to me).**
 
-*A:* Implement new storage that will be managed by the application state to store this information. Implement a new HTTP API, that contains a new method to get this information.
+*A:* Implement new storage that will be managed by the application state (NOTE: item 3 has this as "ApplicationState"; please make consistent) to store this information. Implement a new HTTP API, that contains a new method to get this information. (Should the immediately preceding sentence instead be "Implement a new HTTP API call that contains a method to get this information."?)
```

Comment:
@zennermac 

*A:* Implement a new storage that will be managed by the ApplicationState to store this information. Then implement a new HTTP API that contains a new method to get this information.